expressions.html

```diff
@@ -587,11 +579,17 @@
 			<td></td>
 		</tr>
 		<tr title="Arrays with custom syntax">
```

Comment:
The custom syntax is not part of the test. You're using custom syntax because that's the only way you can test arrays (well, that or `mv-value`), but the test is about arrays, not custom syntax.